README.md

```diff
@@ -27,7 +27,7 @@ Algorithms to
 * shorten them relative to a reference location, and
 * recover a location from a short code and a reference location given as latitude/longitude pair
 
-are publicly available and can be used without restriction. Geocoding services are not a part of the Open Location Code technology.
+Are publicly available and can be used without restriction. Geocoding services are not a part of the Open Location Code technology.
```

Comment:
This is a continuation of the sentence that starts before the bullet points, so you can't just capitalise it without doing something about the previous sentence.